legacy/core/src/main/java/com/fsck/k9/preferences/SettingsImporter.kt

```diff
@@ -100,6 +103,11 @@ class SettingsImporter internal constructor(
                 }
             }
 
+            if (preferences.getAccounts().size > 1) {
```

Comment:
`SettingsImporter` is not only used for importing QR codes. It's primary use is importing settings files. Those can also include general settings (called "global settings" in the code). If the user imports general settings, we want to use the unified inbox setting from there and not overwrite it. 

So this also has to check if `globalSettingsImported` is `false`.

Reply:
Done!